unix.c

```diff
@@ -201,7 +232,7 @@ enet_address_equal (ENetAddress * address1, ENetAddress * address2)
         return sin1 -> sin_port == sin2 -> sin_port &&
             sin1 -> sin_addr.s_addr == sin2 -> sin_addr.s_addr;
     }
-#ifdef AF_INET6
+#if defined(AF_INET6) && !(defined(__3DS__))
```

Comment:
What's missing in the 3DS SDK? Do they define `AF_INET6` but not `struct sockaddr_in6` or something?

Reply:
Yup! I con only really find references to `sockaddr_in6` in devkitArm's "experimental" code, but it's not actually defined in the toolkit.

Reply:
OK, you can revert this part then since you already `#undef AF_INET6` above.

Reply:
Done! Sorry, I must've forgotten to remove those when I added the `undef` bits.